src/compiler/config/outputs/index.ts

```diff
@@ -22,6 +22,9 @@ export const validateOutputTargets = (config: d.Config, diagnostics: d.Diagnosti
       err.messageText = `Invalid outputTarget type "${
         outputTarget.type
       }". Valid outputTarget types include: ${VALID_TYPES.map((t) => `"${t}"`).join(', ')}`;
+    } else if (outputTarget.type === DIST_CUSTOM_ELEMENTS_BUNDLE) {
+      const warning = buildWarn(diagnostics);
+      warning.messageText = `dist-custom-elements-bundle is deprecated and will be removed in a future version. Use "dist-custom-elements" instead. If "dist-custom-elements" does not meet your needs, please add a comment to https://github.com/ionic-team/stencil/issues/3136.`;
```

Comment:
Should we specify this will only occur in a _major_ version?

```suggestion
      warning.messageText = `dist-custom-elements-bundle is deprecated and will be removed in a future major version. Use "dist-custom-elements" instead. If "dist-custom-elements" does not meet your needs, please add a comment to https://github.com/ionic-team/stencil/issues/3136.`;
```

CC @splitinfinities

Reply:
Good catch 👍